docs/source/troubleshooting.rst

```diff
@@ -13,8 +13,37 @@ Whether you are using DCA or schematic API or schematic library/CLI, the followi
 
     1. If you are validating or submitting the manifest, how was the manifest initiatially generated?  If manually and NOT using schematic, there may be errors.
     2. If the manifest was generated by schematic, when was it generated?  Did you download the previously submitted manifest from Synapse and modify it?  Did you download it and resubmit it?  Please run the manifest generate command again to have a fresh manifest.
+4. Does your data model have any "reserved words" as attribute names?
 
-4. Create a Github issue or reach out to your respective DCC service desks.  What is the schematic or DCA configuration used? Specifically, it's most important to capture the following:
+   The following are reserved words that should not be used as attribute names in your data model:
```

Comment:
There are other reserved keywords that are synapse specific. Take a look at entity view reserved reserved column headers